src/debugger/stepper.jl

```diff
@@ -76,8 +77,55 @@ function add_breakpoint(bp)
   JuliaInterpreter.breakpoint(bp["file"], bp["line"], cond)
 end
 
+function debug_file(mod, text, path, should_step)
+  mod = getmodule(mod)
+
+  hideprompt() do
+    local expr, exprs
+    try
+      expr = Base.parse_input_line(text, filename = path)
+      exprs, _ = JuliaInterpreter.split_expressions(mod, expr; filename = path)
+    catch err
+      println(stderr)
+      Base.showerror(stderr, ErrorException("Error while parsing file or runtime error."), [])
+      println(stderr)
+      return
+    end
+
+    lc = should_step ? :stepexpr : :continue
+
+    debugmode(true)
+
+    for (i, (mod, ex)) in enumerate(exprs)
+      lc == :stop && break
+
+      temp_bps = add_breakpoint.(Atom.rpc("getFileBreakpoints"))
```

Comment:
Do we still need this ? Aren't breakpoints added in `startdebugging` anyway for now ?

Reply:
We do, yeah. Not entirely sure why though :P